be/src/exprs/array_functions.tpp

```diff
@@ -1060,4 +1060,300 @@ private:
     }
 };
 
+enum ArithmeticType { SUM, AVG, MIN, MAX };
+
+class ArrayArithmetic {
+public:
+    template <ArithmeticType type, LogicalType value_type, LogicalType sum_result_type, LogicalType avg_result_type,
```

Comment:
suggest to use camel naming instead of snake naming for template type, local variables in functions are already snake style, it seems confusing. 
for examples:  
1. value_type -> ValueType
2. sum_result_type->SumResultType

Reply:
！I agree with you！I don't know who implemented the array function, really confusing... I will fix it